lib/elastic/index.ex

```diff
@@ -31,7 +31,7 @@ defmodule Elastic.Index do
   """
 
   def create(index) do
-    HTTP.post(name(index))
+    HTTP.put(name(index))
```

Comment:
@nicholasjhenry Elasticsearch uses `PUT` for index creation:
[5.3 documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.3/indices-create-index.html), [2.3 documentation](https://www.elastic.co/guide/en/elasticsearch/reference/2.3/indices-create-index.html)

Using `POST` for index creation apparently works with 2.3 (in our setup) but does not work with 5.3.